pkg/apis/operator/v1alpha1/types.go

```diff
@@ -526,6 +529,82 @@ type GardenerSchedulerConfig struct {
 	LogLevel *string `json:"logLevel,omitempty"`
 }
 
+// GardenerDashboardConfig contains configuration settings for the gardener-dashboard.
+type GardenerDashboardConfig struct {
+	// EnableTokenLogin specifies whether it is possible to log into the dashboard with a JWT token. Either this or OIDC
+	// must be configured.
+	// +kubebuilder:default=true
+	// +optional
+	EnableTokenLogin *bool `json:"enableTokenLogin,omitempty"`
+	// FrontendConfigMapRef is the reference to a ConfigMap containing the frontend configuration.
+	// +optional
+	FrontendConfigMapRef *corev1.LocalObjectReference `json:"frontendConfigMapRef,omitempty"`
+	// GitHub contains configuration for the GitHub ticketing feature.
+	// +optional
+	GitHub *DashboardGitHub `json:"gitHub,omitempty"`
+	// LogLevel is the configured log level. Must be one of [trace,debug,info,warn,error].
+	// Defaults to info.
+	// +kubebuilder:validation:Enum=trace;debug;info;warn;error
+	// +kubebuilder:default=info
+	// +optional
+	LogLevel *string `json:"logLevel,omitempty"`
+	// OIDC contains configuration for the OIDC settings.
+	// +optional
+	OIDC *DashboardOIDC `json:"oidc,omitempty"`
```

Comment:
Actually, I now renamed it to `oidcConfig` because this is also [what it's called in the KAPI config](https://github.com/gardener/gardener/blob/c8a2808be0faa75f2d0df2e1be46717fd9245bc8/example/operator/20-garden.yaml#L73-L87). Hope this is fine.